onnxruntime/contrib_ops/cpu/transformers/logits_processor.h

```diff
@@ -141,10 +150,125 @@ class PresencePenaltyLogitsProcessor : public ILogitsProcessor<T> {
 template <typename T>
 class TimestampLogitsProcessor : public ILogitsProcessor<T> {
  public:
-  TimestampLogitsProcessor(int eos_token_id, int max_initial_timestamp_index);
+  TimestampLogitsProcessor(int eos_token_id, int max_initial_timestamp_index)
+      : eos_token_id_(eos_token_id), max_initial_timestamp_index_(max_initial_timestamp_index) {}
 
   void Process(const ISequences* sequences,
-               NextTokenScores<T>& next_token_scores) override;
+               NextTokenScores<T>& next_token_scores) override {
+    const int beg_token_id_ = eos_token_id_ + 107;
+    const int not_token_id_ = eos_token_id_ + 106;
+    const int solm_token_id_ = eos_token_id_ + 105;
+    const int sot_token_id_ = eos_token_id_ + 1;
+    constexpr int translate_token_id_ = 50358;
```

Comment:
Since the translate and transcribe token ids will be updated to support both the multilingual and English-only models in [another PR](https://github.com/microsoft/onnxruntime/pull/17500#discussion_r1346221908), we should add a 'todo' comment to remember this.

Reply:
Added.